base/applications/cmdutils/xcopy/lang/zh-TW.rc

```diff
@@ -1,60 +1,67 @@
-/* Traditional Chinese translation by Henry Tang Ih 2016 (henrytang2@hotmail.com) */
+/*
+ * PROJECT:     xcopy command
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Chinese (Traditional) resource file
+ * TRANSLATORS: Copyright 2016 Henry Tang Ih <henrytang2@hotmail.com>
+ *              Copyright 2021 Chan Chilung <eason066@gmail.com>
+ * REFERENCES:  Wine's Chinese resource translation
+ */
 
 LANGUAGE LANG_CHINESE, SUBLANG_CHINESE_TRADITIONAL
 
 STRINGTABLE
 {
-    STRING_INVPARMS, "無效的參數數目 - 使用 xcopy /? 求助\n"
-    STRING_INVPARM,  "無效的參數 '%1' - 使用 xcopy /? 求助\n"
+    STRING_INVPARMS, "無效的參數數量 - 使用 xcopy /? 以獲得說明\n"
+    STRING_INVPARM,  "無效的參數 '%1' - 使用 xcopy /? 以獲得說明\n"
     STRING_PAUSE,    "請按 <Enter> 開始複製\n"
-    STRING_SIMCOPY,  "%1!d! 將複製檔案\n"
-    STRING_COPY,     "%1!d! 檔案已複製\n"
-    STRING_QISDIR,   "'%1' 的檔名或在目標系統上\n\
-                     的目錄嗎?\n\
+    STRING_SIMCOPY,  "%1!d! 個檔案將被複製\n"
+    STRING_COPY,     "已複製 %1!d! 個檔案\n"
+    STRING_QISDIR,   "%1 是目標上的\n\
+                     一個檔案名稱還是目錄?\n\
                      (F - 檔案, D - 目錄)\n"
     STRING_SRCPROMPT,"%1? (Yes|No)\n"
-    STRING_OVERWRITE,"要覆蓋 %1 嗎? (Yes|No|All)\n"
-    STRING_COPYFAIL, "為 '%2' 的 '%1' 的複製失敗， r/c %3!d!\n"
-    STRING_OPENFAIL, "未能開啟 '%1'\n"
-    STRING_READFAIL, "在 '%1' 的閱讀過程中失敗\n"
+    STRING_OVERWRITE,"是否覆寫 %1? (Yes|No|All)\n"
+    STRING_COPYFAIL, "將 '%1' 複製到 '%2' 時失敗，並獲得 r/c %3!d!\n"
+    STRING_OPENFAIL, "無法開啟 '%1'\n"
+    STRING_READFAIL, "在讀取 '%1' 時發生了失敗\n"
     STRING_YES_CHAR, "Y"
     STRING_NO_CHAR,  "N"
     STRING_ALL_CHAR, "A"
     STRING_FILE_CHAR,"F"
     STRING_DIR_CHAR, "D"
 
     STRING_HELP,
-"XCOPY - 副本的源的檔案或目錄樹到目的地。\n\
+"XCOPY - 將來源檔案或樹狀目錄複製到目的地。\n\
 \n\
 語法:\n\
-XCOPY 源 [目標] [/I] [/S] [/Q] [/F] [/L] [/W] [/T] [/N] [/U]\n\
-\t     [/R] [/H] [/C] [/P] [/A] [/M] [/E] [/D] [/Y] [/-Y]\n\
+XCOPY 來源 [目的地] [/I] [/S] [/Q] [/F] [/L] [/W] [/T] [/N] [/U]\n\
+\t     [/R] [/H] [/C] [/P] [/A] [/M] [/E] [/D] [/Y] [/-Y] [/K]\n\
```

Comment:
just found out the `[\K]` parameter is missed on the English resource file. May I ask whether I need to change it separately from this PR?

Reply:
Yes, better change it separately.

Reply:
OK. Noted with thanks.